data.sql

```diff
@@ -1,14 +1,24 @@
 /* Populate database with sample data. */
 
 insert into animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
-values ('Agumon','2020-02-03',0,true,10.23);
+values ('Charmander','2020-02-08',0,false,-11);
 
 insert into animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
-values ('Gabumon','2018-11-18',2,true,8);
+values ('Plantmon','2021-11-15',2,true,-5.7);
 
 insert into animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
-values ('Pikachu','2021-01-07',1,false,15.04);
+values ('Squirtle','1993-04-02',3,false,-12.13);
 
 insert into animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
-values ('Devimon','2017-05-12',5,true,11);
+values ('Angemon','2005-06-12',1,true,-45);
+
+insert into animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+values ('Boarmon','2005-06-07',7,true,-20.4);
+
+insert into animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+values ('Blossom','1998-10-13',3,true,-17);
+
+insert into animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+values ('Ditto','2022-05-14',4,true,-22);
```

Comment:
- [ ] You did well changing the data values to the correct ones based on the project requirement. But it appears you introduced some error in the `weight_kg` value of `Boarmon`, `Blossom`, and `Ditto`. These ones are not negative values as you have made here. Kindly correct.